templates/systemd/etc/systemd/system/sshnpd.service

```diff
@@ -8,7 +8,7 @@ Type=simple
 Restart=always
 ExecStartPre=/bin/sleep 10
 RestartSec=3
-ExecStart=/usr/bin/screen -DmS sshnpd /home/pi/sshnp/sshnpd -a <@deviceatsign> -m <@manageratsign> -u -d <my_device> -s -v
+ExecStart=/usr/bin/screen -DmS sshnpd /home/pi/.local/bin/sshnpd -a <@client_atsign> -m <@device_manager_atsign> -u -d <device_name> -s -v
```

Comment:
Should we have `pi` as the user here, or something more generic?

Reply:
Changing it to %h which should specify the home dir https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Specifiers